abdiff/core/__init__.py

```diff
@@ -5,5 +5,6 @@
 
 from abdiff.core.init_job import init_job
 from abdiff.core.init_run import init_run
+from abdiff.core.run_ab_transforms import run_ab_transforms
 
-__all__ = ["init_job", "init_run"]
+__all__ = ["init_job", "init_run", "run_ab_transforms"]
```

Comment:
Missed in my first pass, but +1 to this pattern!  I think it will be nice to do `from abdiff.core import XYZ` for all the main core functions.